packages/zarf-agent/zarf.yaml

```diff
@@ -7,9 +7,9 @@ components:
   - name: zarf-agent
     required: true
     images:
-      - "defenseunicorns/zarf-agent:v0.19.5"
+      - "defenseunicorns/zarf-agent:992efd"
```

Comment:
I know we are going to change how we do agent versioning in the future when we can add values at build/run time. 

How is the versioning going to work in the mean time?

Reply:
I'm just changing to the commit hash it was built from so it's more accurate for now.

Reply:
Personally I think this is fine for now (it is more traceable this way).  If we do change agent versioning soon then we can address that without much user impact (would just be an updated version to them).